honeycomb-core/src/cmapbuilder/grid/building_routines.rs

```diff
@@ -11,6 +11,7 @@ pub fn build_2d_grid<T: CoordsFloat>(
     [n_square_x, n_square_y]: [usize; 2],
     [len_per_x, len_per_y]: [T; 2],
 ) -> CMap2<T> {
+    #[allow(deprecated)] // allow because CMap2::new() will still be available inside the crate
```

Comment:
Why do you not call another nonpublic new constructor (to which the old `new()` aliases) ?
This way, you can get rid of all these `allow(deprecated).`

Reply:
I didn't think of making the change then, I fixed this is #90